### PR TITLE
161 workover chapter 71 introduction

### LINKED ISSUE #160: Add reference to other ASAM standards
**Is your feature request related to a problem? Please describe.**
Currently no relation to other ASAM standards is described.

**Describe the solution you'd like**
Add documentation about how to use OpenMATERIAL in combination with e.g. OSI oder OpenSCENARIO.
- Use ASAM OSI model_reference field to reference a 3D model file or an asset file with a relative or absolute path (to be discussed)
- Use ASAM OpenSCENARIO SceneGraphFile field to reference the 3D model file or an asset file of a stationary environment
- Use ASAM OpenSCENARIO model3d field to reference the 3D model file or an asset file of a moving object (e.g. in the vehicle catalog)

### LINKED ISSUE #161: Workover Chapter 7.1: Introduction
**Is your feature request related to a problem? Please describe.**

* Chapter 7.1.7 is obsolete
* Chapter 7.1.8 has to be updated based on #155 and moved to 7.2.5 -> A section to explain the example, add comment "Color codes have to be consistent througout all OpenMATERIAL textures."
* Headlines are too long
* Bullet points in 7.1.5 are not necessary
* Bullet points in 7.1.2 and  7.1.6 are not properly formatted

**Describe the solution you'd like**

1. Fix the above
2. Ask project group for review

**Describe alternatives you've considered**
-

**Additional context**
-

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
## Describe your changes

- [x] Restructure geometry chapter introduction
- [x] Editorial changes
- [x] Added interaction with other ASAM standards

## Issue ticket number and link

Fixes #161, fixes #160

## Checklist before requesting a review

- [x] I have performed a self-review of my code/documentation.
- [x] My changes generate no new warnings during the documentation generation.

### CODE REVIEW

**Comment by ClemensLinnhoff** [comment]
@FKlopfer could you do an editorial review?

File: content/geometry/introduction.adoc [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
-* Open Simulation Interface handles sensor data exchanging with other parties in the simulation, which is influenced by the material characteristics defined by OpenMATERIAL.
-https://www.asam.net/standards/detail/osi/
+== Interaction with other ASAM standards
 
-==    Is the standardization of 3D models specified by OpenMATERIAL separated from the standardization of materials in OpenMATERIAL? 
-NOTE: TODO: (tbd)
+The ASAM OpenX standards aim for comprehensive, realistic and most of all exchangeable simulation data for the development and testing of autonomous systems.
 
-==    How are materials referenced (brief introduction and reference to dedicated chapter here) (concrete description and chapter tbd)
+* ASAM OpenDRIVE standardizes the description of road networks. Is contains some geometric definitions of the road surface, but mainly targets a semantic description of the road.
```
Comment (AsamDiegoSanchez) [inline content/geometry/introduction.adoc:33]: Typo: "It" contains some ...

**Review by AsamDiegoSanchez — APPROVED** [review]
It looks good. I found just a minor typo

File: content/geometry/introduction.adoc (hunk 1 above)
Comment (ClemensLinnhoff) [reply content/geometry/introduction.adoc]: Thanks, I fixed it.

**Comment by FKlopfer** [comment]
Editorial review done.

**Review by FKlopfer — APPROVED** [review]

**Pull request merged**